.github/workflows/mobile-android_build.yml

```diff
@@ -149,7 +149,7 @@ jobs:
             --config=mobile-remote-release-clang-android
             //test/kotlin/apps/experimental:hello_envoy_kt
           steps-post: |
-            - uses: envoyproxy/toolshed/gh-actions/envoy/android/post@actions-v0.2.15
+            - uses: envoyproxy/toolshed/gh-actions/envoy/android/post@actions-v0.2.17
```

Comment:
Is it possible to define this version somewhere, and just use the version throughout?  That way in the future, we'd only have to update in one place?  That can also be done in a follow-up if you're just trying to get this to land

Reply:
unfortunately not - at least in the main or i would defo have done that

in this specific case it is possible just because its a "dynamic" step - but that is not the case in 99% of the usage